### PR TITLE
Basque translation

### PR DESCRIPTION
**Pull request by izendegi**
Hi Nicolas,

Here goes the Basque translation for the Moodle Tool Guide [as I promised](https://twitter.com/izendegi/status/1273278002823135234).

I've browsed the repo looking for content to translate, but please tell me if there is something missing/wrong (I'm not sure about the changes made to the config.yml file).

I hope everything is OK, but once I see it rendered I may fix some texts if they are too long.

Cheers!

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -135,3 +135,28 @@ languages:
...
+    title: Moodleko Tresnen Gida
+    slogan: Moodleko Tresnen Gida irakasle eta hezitzaileentzat
+    description: 'Moodleko Tresnen Gida: 3 edo bertsio berriago bat erabiltzeko irakasle, hezitzaile eta trebatzaileentzat diseinatutako erreferentziazko taula da.'
+    contentDir: content/eu
+    date_format: 2006 January 2
+    copyright: '[Nicolas Martignoni](https://blog.martignoni.net/a-propos/)-ek egina – Webgunea __[moodletoolguide.net](https://moodletoolguide.net)__ [Creative Commons BY-NC-SA 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/) lizentziapean argitaratu da.
+
+
+      Tresna hau [Eastern Institute of Technology](https://www.eit.ac.nz/)-ko (Zeelanda Berria) [Joyce Seitzinger](https://twitter.com/catspyjamasnz)-en ideia izan zen, 2020ko maiatzean Moodle 3.x-erako [Nicolas Martignoni][nm]-eko egokitua [Gavin Henrick](https://twitter.com/ghenrick)-ek 2 bertsiorako egindako lanean oinarrituta.'
+    weight: 1
```
Comment (martignoni) [inline config.yml:150]: Weight should be 5

File: content/eu/about/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Honi buruz
+description: Zer da Moodleko Tresnen Gida?
+---
+
+_Moodleko Tresnen Gida_ [Moodle][moodle] 3 edo bertsio berriago bat erabiltzeko irakasle, hezitzaile eta trebatzaileentzat diseinatutako erreferentziazko taula da. Moodleren tresna nagusiei buruzko informazioa ematen du, eta lortu nahi den helburu pedagogikoaren arabera tresnarik egokienak aukeratzen laguntzen du. Tresna bakoitza erabiltzean lor daitezkeen [Bloom-en maila taxonomikoak][bloom] ere aipatzen ditu.
+
+Tresna hau [Eastern Institute of Technology](https://www.eit.ac.nz/)-ko (Zeelanda Berria) [Joyce Seitzinger](https://twitter.com/catspyjamasnz)-en ideia izan zen, 2020ko maiatzean Moodle 3.x-erako [Nicolas Martignoni][nm]-eko egokitua [Gavin Henrick](https://twitter.com/ghenrick)-ek 2 bertsiorako egindako lanean oinarrituta.
```
Comment (martignoni) [inline content/eu/about/index.md:8]: This line should be a translation of "An idea by Joyce Seitzinger of the Eastern Institute of Technology (NZ), updated in May 2020 for Moodle 3.x by Nicolas Martignoni and translated into Basque by Iñigo Zendegi."

File: config.yml [hunk 3]
```diff
@@ -135,3 +135,28 @@ languages:
...
+    languageName: Euskara
+    title: Moodleko Tresnen Gida
+    slogan: Moodleko Tresnen Gida irakasle eta hezitzaileentzat
+    description: 'Moodleko Tresnen Gida: 3 edo bertsio berriago bat erabiltzeko irakasle, hezitzaile eta trebatzaileentzat diseinatutako erreferentziazko taula da.'
+    contentDir: content/eu
+    date_format: 2006 January 2
+    copyright: '[Nicolas Martignoni](https://blog.martignoni.net/a-propos/)-ek egina – Webgunea __[moodletoolguide.net](https://moodletoolguide.net)__ [Creative Commons BY-NC-SA 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/) lizentziapean argitaratu da.
+
+
+      Tresna hau [Eastern Institute of Technology](https://www.eit.ac.nz/)-ko (Zeelanda Berria) [Joyce Seitzinger](https://twitter.com/catspyjamasnz)-en ideia izan zen, 2020ko maiatzean Moodle 3.x-erako [Nicolas Martignoni][nm]-eko egokitua [Gavin Henrick](https://twitter.com/ghenrick)-ek 2 bertsiorako egindako lanean oinarrituta.'
```
Comment (martignoni) [inline config.yml:149]: Please only translate this This line should be a translation of "An idea by Joyce Seitzinger of the Eastern Institute of Technology (NZ), updated in May 2020 for Moodle 3.x by Nicolas Martignoni and translated into Basque by Iñigo Zendegi.", obviously with the links.

See below too.

**Comment by martignoni** [comment]
Could you please also translate the keywords?
```
 keywords:
 - Moodle
 - Moodle Tools
 - Moodle Tools Guide
 - Moodle Tool Guide
 - Moodle Tool Guide for teachers
 - Moodle Tools for teachers
 - Moodle Guide
 - Moodle Guide for teachers
 - Moodle Guide for instructors
 - Moodle Guide for trainers
```

File: content/eu/about/index.md (hunk 2 above)
Comment (izendegi) [reply content/eu/about/index.md]: I've updated that text both here and in config.yml

File: config.yml (hunk 3 above)
Comment (izendegi) [reply config.yml]: I've updated that text both here and in content/eu/about/index.md

**Comment by izendegi** [comment]
Hi Nicolas,

I've already comitted the proposed changes. 

And regarding those keywords, the translation would be this:

 - Moodle --> Moodle
 - Moodle Tools --> Moodleko Tresnak
 - Moodle Tools Guide --> Moodleko Tresnen Gida
 - Moodle Tool Guide --> Moodleko Tresnen Gida
 - Moodle Tool Guide for teachers --> Moodleko Tresnen Gida irakasleentzat
 - Moodle Tools for teachers --> Moodleko Tresnak irakasleentzat
 - Moodle Guide --> Moodleko Gida
 - Moodle Guide for teachers --> Moodleko Gida irakasleentzat
 - Moodle Guide for instructors --> Moodleko Gida hezitzaileentzat
 - Moodle Guide for trainers --> Moodleko Gida trebatzaileentzat

**Comment by martignoni** [comment]
Your translation is published. A big thank you.

**Comment by izendegi** [comment]
Great, thank you too!

I'll review all the translation now I can see it rendered and will send another PR if I see any typo or anything that can be improved.

**Comment by martignoni** [comment]
Thanks. Don't forget to [update your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork) before sending the new PR.

**Pull request merged**